### PR TITLE
ADR 028 for using Shield Advanced

### PR DESCRIPTION
**Pull request by sarahseewhy**
# PR Checklist

- [x] If you are proposing a new decision record document, used the right template for that
      - ([ADR](https://github.com/alphagov/forms/blob/main/ADR/ADRXXX-architecture-decision-record-template.md), [decision-record](https://github.com/alphagov/forms/blob/main/decision-record/DRXXX-decision-record-template.md), engagement, [research](https://github.com/alphagov/forms/blob/main/research/YYYY-MM-DD-template.md))
- [x] Set yourself as the Assignee
- [x] Tag anyone you would like to review, or @forms-design or @forms-devs
- [x] Fill in the template below

---

# What

Add a new ADR capturing the context, decision, and pros/cons of using AWS Shield Advanced in our infrastructure.

**I've written a sparse ADR because I'm conscious this is a security-related decision on a public-facing repo.**

If we feel it's sensible and secure for me to go into further detail about consequences and/or implementation I'm happy to do so. 

# How to review

Describe the steps required to test the changes.

1. Semantic: Do you agree with the changes? What's the balance between security and context?
1. Syntactic: Spelling, grammar, etc.

# Who can review

Anyone except @sarahseewhy (me) because I worked on it.

### CODE REVIEW

**Comment by sarahseewhy** [comment]
> 👍 looks good and I think sensible for a public facing repo
> 
> There seem to be a couple of commits that don't belong to this PR

Thanks! 

I'm also a bit perplexed by the other commits. I'm trying to figure out how this happened, whether I made a mistake while branching from `main` or whether we were all working on the same branch by accident 🤷

**Comment by sarahseewhy** [comment]
> There seem to be a couple of commits that don't belong to this PR

I'm still unsure how this happened. The changes proposed in the `service-map` file were already on the `main` branch. Perhaps the file was edited in the UI and it somehow got included in my PR?

I dropped the commits and the file is no longer part of the PR nor are the commits.

File: ADR/ADR028-use-shield-advanced.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# ADR028: Use AWS Shield Advanced
```
[the remark this answers is not in the document]
Comment (alice-carr) [reply ADR/ADR028-use-shield-advanced.md:1]: other than that it looks great - happy to approve once the number is updated

[the remark this answers is not in the document]
Comment (sarahseewhy) [reply ADR/ADR028-use-shield-advanced.md:1]: I think there's an ADR 29 already? https://github.com/alphagov/forms/pull/153

[the remark this answers is not in the document]
Comment (sarahseewhy) [reply ADR/ADR028-use-shield-advanced.md]: Done :)

**Review by alice-carr — APPROVED** [review]

**Pull request merged**